halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/google/GoogleEditBakeryDefaultsCommand.java

```diff
@@ -45,6 +45,12 @@ protected String getProviderName() {
   private String network;
 
   @Parameter(
+      names = "--network-project-id",
```

Comment:
Why `-id`? Everywhere else we just refer to it as `project`, in component config & halyard commands.